group_vars/location_vaterhaus/snmp.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+snmp_devices:
+  - hostname: vaterhaus-zwingli
+    address: 10.230.192.202
+    snmp_profile: af60
+
+snmp_devices:
+  - hostname: vaterhaus-w
+    address: 10.230.192.203
+    snmp_profile: airos_8
+
+#snmp_devices:
+#  - hostname: vaterhaus-no
+#    address: 10.230.192.204
+#    snmp_profile: mikrotik
+
+snmp_devices:
+  - hostname: vaterhaus-o
+    address: 10.230.192.205
+    snmp_profile: airos_8
+
+snmp_devices:
+  - hostname: vaterhaus-adlershof
+    address: 10.230.192.206
+    snmp_profile: airos_8
+
+snmp_devices:
+  - hostname: vaterhaus-cg47
+    address: 10.230.192.207
+    snmp_profile: airos_8
+
+#snmp_devices:
```

Comment:
i think that device wont ever have snmp working. therefore the comment can be deleted